### ISSUE TITLE
add a unit converter between Klingon and Terran units

### ISSUE THREAD

**Issue by De7vID** [De7vID/klingon-assistant#486]
The request is to add a built-in converter into the apps (Android, iOS) which can go between Klingon units with known values and their common Terran equivalents: **'uj**, **cheb**, **SImyon**, **qelI'qam**, **morgh**, etc.

**Comment by LievenLitaer** [comment]
I'm not saying it's bad, but I wonder if it doesn't go beyond the scope of the boQwI' app? Depending on how large the code is, maybe it would be better to have that as a separate app. (although I have no idea how difficult it is to add an app to the play store.) Nevertheless, I agree that it would be a nice to have.

**Comment by De7vID** [comment]
Potentially related to issue #323, in that maybe someone might want to convert between radix-3 and base-10 numbers at the same time that they want to convert units.

**Comment by De7vID** [comment]
I don't think it goes beyond the scope of the **boQwI'** app. Also, the code would be relatively small, compared to the size of the database, so that isn't really an issue. The main issue is finding someone with both the skills (which I have) and the time (which I do not) to write the necessary code.

**Comment by valdr** [comment]
You're doing a great service as it is, @De7vID! I wish I had the skills...
